testing/bobcat/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=bobcat
+pkgver=4.07.00
+pkgrel=0
+pkgdesc="library of C++ classes and templates handling child processes, streams/sockets, shared memory, config files, etc."
```

Comment:
Could you please make it a bit shorter, less than 100 chars? And start with a capital letter.